bin/server

```diff
@@ -1,6 +1,8 @@
 #!/usr/bin/env node
+
 'use strict';
 
+/* eslint-disable import/no-dynamic-require */
```

Comment:
Put this next to the `require` that specifically triggers this error, and remember to re-enable it immediately after.